lib/core_extensions.rb

```diff
@@ -39,8 +39,14 @@ def initialize(*attribute)
 
     def before_destroy(record)
       klasses.each do |klass|
-        record.send(klass.to_sym).each do |what|
-          what = what.to_label unless what.is_a? String
+        record.association(klass.to_sym).association_scope.each do |what|
+          unless what.is_a? String
+            if what.class.find_by_id(what.id)
```

Comment:
Yeah, `example.com is being used by a hidden %s resource` (or "out of scope %s resource").  Use _() around the class name, since we have some human-friendly translations available.